android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -28,6 +30,9 @@
 public class LogTest extends BaseTest {
     private Context context;
 
+    @Rule
+    public TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
We should make it private.

Reply:
What is @Rule?

Reply:
@Rule is an annotation the enables kind of Aspect Oriented trickery.  Fortunately, we don't use it.  It should be removed.

Reply:
This will help us create temporary random folder creation rather than using the manual random folder. 
https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html

Reply:
we should make it `private` and `final`